Userland/Utilities/shot.cpp

```diff
@@ -120,18 +120,15 @@ int main(int argc, char** argv)
         app->exec();
 
         crop_region = container.region();
-        if (crop_region.is_empty()) {
+        if (crop_region.has_value()) {
             dbgln("cancelled...");
             return 0;
         }
     }
 
     sleep(delay);
-    Optional<u32> screen_index;
-    if (screen >= 0)
-        screen_index = (u32)screen;
     dbgln("getting screenshot...");
-    auto shared_bitmap = GUI::WindowServerConnection::the().get_screen_bitmap(crop_region, screen_index);
+    auto shared_bitmap = GUI::WindowServerConnection::the().get_screen_bitmap(crop_region, screen);
```

Comment:
Maybe I'm misreading this but are you now always passing a filled `Optional` which means in `ClientConnection::get_screen_bitmap` we always take the first branch.
Does that not change the behavior if no screen parameter is given?

~(If this is intentional then perhaps the `get_screen_bitmap` function should be cleaned up)~ Nevermind it has other users.

Reply:
Hmm... You are right, before it would take a screenshot of two screens together. I will revert this change.